appserver/orb/orb-iiop/src/main/java/org/glassfish/enterprise/iiop/impl/IIOPSSLSocketFactory.java

```diff
@@ -105,10 +106,9 @@
     private static final String PERSISTENT_SSL = "PERSISTENT_SSL";
 
     private static final int BACKLOG = 50;
-    
+
     private static final String SO_KEEPALIVE = "fish.payara.SOKeepAlive";
```

Comment:
Same here. Suggest property to be ```fish.payara.so-keepalive``` to prevent case sensitivity issues

Reply:
Changed to `fish.payara.SO_KEEPALIVE` as this is the most similar to how the property is used elsewhere.